### PR TITLE
migrated a lot of custom css over to semantic for better scalability,…

### PR DESCRIPTION
**Pull request by k-cleary**
… also added nav to header
Did a lot of tweaking in this PR. Finally added nav, also think I may have figured out an elegant way to keep clear of login clutter (dropdown menu!)

### CODE REVIEW

**Comment by JesseTG** [comment]
Did you update your branch with the latest changes from `master`?

**Comment by k-cleary** [comment]
will do this... now... and reissue request :)

**Comment by JesseTG** [comment]
There's no need to redo the PR unless your commit history gets fucked up, which it's not.

**Comment by k-cleary** [comment]
omg I'm really dumb, but I figured out what I needed to do so it should be all set now (I pulled in changes from master and managed merge conflicts) I am so dumbbbbb <3

File: src/main/java/conf/messages.properties [hunk 1]
```diff
@@ -21,9 +21,9 @@ header.new=New
 header.login=Login
 header.logout=Logout
 header.signup=Signup
-header.facebook=facebook
-header.google=google
-header.deviantart=deviantart
+header.facebook=Facebook
+header.google=Google
+header.deviantart=Deviant Art
```
Comment (Bsabz) [inline src/main/java/conf/messages.properties]: Maybe this is what is messing up the Oauth stuff?

File: src/main/java/www/purple/mixxy/views/layout/header.ftl.html [hunk 2]
```diff
@@ -1,15 +1,47 @@
...
+                                        <a class="item">Profile</a>
+                                        <a class="item" href="/logout">${i18n("header.logout")}</a>
+                                    </div>
+                                    <#else>
+                                        <p>Sign In
+                                            <i class="dropdown icon"></i></p>
+                                        <div class="menu">
+                                            <a class="item" href="/login?provider=facebook" id="facebook">${i18n("header.facebook")}</a>
+                                            <a class="item" href="/login?provider=google" id="google">${i18n("header.google")}</a>
+                                            <a class="item" href="/login?provider=deviantart" id="deviantart">${i18n("header.deviantart")}</a>
```
Comment (Bsabz) [inline src/main/java/www/purple/mixxy/views/layout/header.ftl.html]: The header.fb,google,deviantart are not the same id's at mentioned in the java/conf/message.properties file

**Comment by Bsabz** [comment]
yay!

**Pull request merged**